api/agentpb/agent.proto

```diff
@@ -310,6 +310,8 @@ message CheckConnectionResponse {
   // Error message if connection check failed.
   string error = 1;
   Stats stats = 2;
+  // Contains the body of HTTP response of connection check to an exporter.
```

Comment:
to be honest it looks weird for me that all other connections we check on pmm-agent side (even returning table_count), but for exporters we move connection check to pmm-server side.